docker-compose.yaml

```
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
+    command: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)